app/views/accounts/_validated_badge.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if @presenter.show_verified_badge? %>
+  <%= render BadgeTooltipComponent.new(icon: :check_circle, icon_class: 'text-success', tooltip_text: @presenter.biometric_identity_verification? ? t('components.badge_tooltip_component.verified_biometric') : t('components.badge_tooltip_component.verified'), class: 'usa-tooltip', data_position: 'top').with_content(t('headings.account.verified_account')) %>
+<% end %>
+<% if @presenter.show_unverified? %>
+  <%= render BadgeTooltipComponent.new(icon: :check_circle, icon_class: 'text-warning', tooltip_text: t('components.badge_tooltip_component.unverified'), class: 'usa-tooltip border-warning', data_position: 'top').with_content(t('headings.account.unverified')) %>
+<% end %>
+<% if @presenter.show_ipp_partial? || @presenter.show_gpo_partial? %>
+  <%= render BadgeTooltipComponent.new(icon: :check_circle, icon_class: 'text-info', tooltip_text: t('components.badge_tooltip_component.pending'), class: 'usa-tooltip border-info', data_position: 'top').with_content(t('headings.account.pending')) %>
```

Comment:
The designs have this badge using an "Info" button, which makes sense given the color we're trying to apply.

With using the info icon, I think we could move all of the color handling to happen inside `BadgeComponent`, rather than expect that to be passed externally as additional CSS classes. This would make the component easier to use.

```rb
def color_token
  case icon
  when :check_circle
    'success'
  when :warning
    'warning'
  else
    'info'
end

def border_css_class
  "border-#{color_token}"
end

def icon_css_class
  "text-#{color_token}"
end
```

Reply:
> With using the info icon, I think we could move all of the color handling to happen inside `BadgeComponent`, rather than expect that to be passed externally as additional CSS classes. This would make the component easier to use.

Updated in 53e440976f.